docs/controller.md

```diff
@@ -29,6 +29,13 @@
     - **version**: `str` (1.9.4, 2.4.0., etc.) - default is the latest TT
     - **wipe**: `bool` ... whether to delete the emulator profile before starting it - default is False
 
+- **emulator-start-from-url**
+  - **action**: downloads emulator from specified URL and runs it
+  - **arguments**:
+    - **url**: `str` from where to download the emulator
+    - **one_or_two**: `str` which emulator it is - either "1" for T1 or "2" for T2
```

Comment:
Let's use "model" instead?

Reply:
I agree, let's rename it. Also, we can consider what is the best values it could take - it can be strings `"1"` and `"2"`, integers `1` and `2`, strings `"T1"` and `"T2"/"TT"` etc. Current situation is compatible with the first character in version -  `1.2.3` or `2.3.4`

Reply:
Renamed in 430eec3